builtInFunctions/esdtMetaDataRecreate.go

```diff
@@ -109,8 +111,7 @@ func checkUpdateArguments(
 		return ErrInvalidNumberOfArguments
 	}
 
-	tokenId := append([]byte(baseESDTKeyPrefix), vmInput.Arguments[tokenIDIndex]...)
-	return rolesHandler.CheckAllowedToExecute(acntSnd, tokenId, []byte(role))
+	return rolesHandler.CheckAllowedToExecute(acntSnd, vmInput.Arguments[tokenIDIndex], []byte(role))
```

Comment:
is this change ok?

Reply:
Yes, the roles are saved at the tokenId, without the `ELRONDesdt` prefix. Right @sasurobert ?